I've just extended this pull request with an extra commit, marking Go versions 1.17 to 1.19 as using Unicode 13.

Otherwise, with Go 1.19, one would get something like this:

```
# github.com/apparentlymart/go-textseg/v13/autoversion/textseg [github.com/apparentlymart/go-textseg/v13/autoversion/textseg.test]
src/github.com/apparentlymart/go-textseg/v13/autoversion/textseg/version_specific_test.go:15:69: undefined: UnicodeMajorVersion
src/github.com/apparentlymart/go-textseg/v13/autoversion/textseg/version_specific_test.go:18:40: undefined: ScanGraphemeClusters
FAIL	github.com/apparentlymart/go-textseg/v13/autoversion/textseg [build failed]
```

--- a/autoversion/textseg/unicode13.go
+++ b/autoversion/textseg/unicode13.go
@@ -1,4 +1,4 @@
-// +build go1.16,!go1.17
+// +build go1.16,go1.17,go1.18,go1.19,!go1.20
 
 package textseg
 

